kv/variables.go

```diff
@@ -57,6 +60,7 @@ func NewVariables(killed *uint32) *Variables {
 		BackoffLockFast: DefBackoffLockFast,
 		BackOffWeight:   DefBackOffWeight,
 		Killed:          killed,
+		EnableTxnFile:   true,
```

Comment:
it’s better to use [GLOBAL variables](https://docs.pingcap.com/tidb/stable/system-variables) to control whether txn file transaction is enabled or not.
It allows user to try it out on some clusters first. We have `serverless version` in TiDB which is very convenient to turn on all switches.
You can see `serverless version` in 
https://github.com/tidbcloud/tidb-cse/blob/release-7.1-keyspace/session/bootstrap_serverless.go

Reply:
Yes it's designed to be used for variables. And here is the client-go part.

The tidb-cse part will be added after [here](https://github.com/tidbcloud/tidb-cse/blob/77d402036803ea1497a7950b3688e0256627a325/sessionctx/variable/sysvar.go#L1805):

```go
	{Scope: ScopeGlobal | ScopeSession, Name: TiDBBackoffLockFast, Value: strconv.Itoa(tikvstore.DefBackoffLockFast), Type: TypeUnsigned, MinValue: 1, MaxValue: math.MaxInt32, SetSession: func(s *SessionVars, val string) error {
		s.KVVars.BackoffLockFast = tidbOptPositiveInt32(val, tikvstore.DefBackoffLockFast)
		return nil
	}},
	{Scope: ScopeGlobal | ScopeSession, Name: TiDBBackOffWeight, Value: strconv.Itoa(tikvstore.DefBackOffWeight), Type: TypeUnsigned, MinValue: 0, MaxValue: math.MaxInt32, SetSession: func(s *SessionVars, val string) error {
		s.KVVars.BackOffWeight = tidbOptPositiveInt32(val, tikvstore.DefBackOffWeight)
		return nil
	}},
```